scripts/gen_av1_argon.py

```diff
@@ -194,12 +211,12 @@ def generate(self, download: bool) -> None:
                         "test85818": OutputFormat.YUV422P10LE,
                     }
 
-                    if tv_filename in exceptions_profile1_output_format.keys():
+                    if tv_filename in exceptions_profile0_output_format.keys():
+                        output_format = exceptions_profile0_output_format[tv_filename]
+                    elif tv_filename in exceptions_profile1_output_format.keys():
                         output_format = exceptions_profile1_output_format[tv_filename]
                     elif tv_filename in exceptions_profile2_output_format.keys():
                         output_format = exceptions_profile2_output_format[tv_filename]
-                    else:
-                        raise error
```

Comment:
@rsanchez87 i had to remove this because it is causing failures for all test vectors from error folders.